### PR TITLE
Update the plan mentions into the name of the new paid plan.

### PR DESCRIPTION
**Pull request by southp**
#### Changes proposed in this Pull Request

This PR implements the plan mention updates according to the latest marketing audit done for the plan overhaul project. For more details, please refer to p2-pdgrnI-nr. It is created ahead so we have time to finish up translation before launching.

Most of the copies updated here are trivial: just replacing the legacy paid plans as the Managed plan. However, whenever it involves upselling, we need to be cautious that the Managed plan will only have an annual term at the first phase. Thus, this PR also introduces the logics of upselling the Managed plan only for annual plans. 

In order to make it testable on calypso.live, this PR also enables `plans/managed-plan` in `wpcalypso`, `staging` and `production` environment. It also means that **deploying this PR will be the trigger for launching the project to production**. It's not perfect to merge two purposes in one PR like this. However, doing this prevents us from introducing countless if-else conditions to show different copies.

#### Testing instructions

Eventually I will add a more comprehensive test plan here. What's important now is to confirm if the plan mentions are updated correctly.

### CODE REVIEW

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7175355

Hi @southp, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include all of the following strings:
- `Along with all the tools and features built right into WordPress.com, the Pro plan allows you to install other plugins.`
- `When you want to build a one-of-a-kind website, it's time for WordPress.com Pro: upload plugins and themes to create a truly tailored experience for your visitors.`
- `Sites on the Pro Plan using custom plugins and/or custom themes now have the option to switch PHP versions.`
- `All WordPress.com advertising has been removed from your site. Upgrade to Pro to remove the WordPress.com footer credit.`
- `Enrich your posts and pages with video or audio. Upload plenty of media, directly to your site — the Pro Plan has 50 GB storage.`
- `Pro plan required`
- `If you purchased %(domain)s elsewhere, you can {{a}}connect it{{/a}} with WordPress.com Pro.`
- `Only .blog domains are included with your plan, to use a different tld upgrade to a Pro plan.`
- `Upgrade to a Pro plan to unlock the power of our SEO tools!`
- `Unlock powerful SEO tools! Contact your site's administrator to upgrade to a Pro plan.`
- `Use this theme at no extra cost on our Pro Plan`
- `Only site administrators can upgrade to the Pro plan.`
- `Upgrade to Pro Plan`
- `Also included in the Pro Plan`
- `Pay an {{strong}}extra %(extraToPay)s{{/strong}} for our Pro plan, and get access to all its features, plus the first year of your domain for free.`
- `{{strong}}Save %(savings)s{{/strong}} when you purchase a WordPress.com Pro plan instead — your domain comes free for a year.`
- `Purchase our Pro plan at {{strong}}no extra cost{{/strong}}, and get access to all its features, plus the first year of your domain for free.`
- `Did you know that WordPress.com Pro plan customers can now upload any WordPress plugins they want?`
- `And access to the plugin library is just one of many benefits included with the Pro plan.`
- `With our Pro plan you can upload any custom theme you want, have full access to your site with SFTP, and have full control of your data directly with phpMyAdmin.`
- `{{b}}But please note:{{/b}} you don’t currently have access to all these amazing features because they are only available to Pro plan customers and you haven’t upgraded yet.`
- `Upgrade to the Pro plan and start earning`
- `Available only with the Pro plan.`
- `Upgrade to the Pro plan`
- `Upgrade to the Pro plan to access all hosting features`
- `No ads with WordPress.com Pro`
- `Discover an easy way to advertise your brand across Facebook and Instagram. Capture website actions to help you target audiences and measure results. {{em}}Available on Pro plan{{/em}}.`
- `Your current plan doesn't allow plugin installation. Please upgrade to Pro plan first.`
- `%d file could not be uploaded because your site does not support video files. Upgrade to a Pro plan for video support.`
- `Contact your site administrator and ask them to upgrade this site to WordPress.com Pro.`
- `Upgrade to the Pro Plan to Enable VideoPress`
- `Import your entire site with the Pro Plan.`
- `A Pro Plan is required to import everything.`
- `Yes! With the WordPress.com Pro plan you can install any theme you'd like. All plans give you access to our {{a}}directory of free and premium themes{{/a}}. These are among the highest-quality WordPress themes, hand-picked and reviewed by our team.`
- `Yes! With the WordPress.com Pro plan you can install any theme you'd like. All plans give you access to our {{a}}directory of free themes{{/a}}. These are among the highest-quality WordPress themes, hand-picked and reviewed by our team.`
- `Yes! With the WordPress.com Pro plan you can search for and install external plugins. All plans already come with a custom set of plugins tailored just for them. {{a}}Find out more about plugins{{/a}}.`
- `Included in the Pro plan (%s):`
- `Upgrade to the Pro plan to install plugins.`
- `No Ads with WordPress.com Pro`
- `Available with Pro plan`
- `Comes built-in with WordPress.com Pro plan.`
- `Available on Pro plan`
- `Remove the footer credit entirely with WordPress.com Pro`
- `Upload Audio with WordPress.com Pro`
- `Access this theme for FREE with a Pro plan!`
- `Unlock ALL premium themes with our Pro plan!`
- `Upload your own themes with our Pro plan!`
- `Upgrade to the Pro plan to access the theme install features`
- `Requires a Pro plan.`
- `You'll need the Pro plan.`
- `Requires a {{a}}Pro plan{{/a}}`
- `Grow your business with <strong>WordPress.com Pro</strong>`
- `Monetize your site with <strong>WordPress.com Pro</strong>`
- `By upgrading to the Pro plan, you'll enable audio upload support on your site.`

Thank you in advance!

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7175355

Hi @southp, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include all of the following strings:
- `Along with all the tools and features built right into WordPress.com, the Pro plan allows you to install other plugins.`
- `When you want to build a one-of-a-kind website, it's time for WordPress.com Pro: upload plugins and themes to create a truly tailored experience for your visitors.`
- `Sites on the Pro Plan using custom plugins and/or custom themes now have the option to switch PHP versions.`
- `All WordPress.com advertising has been removed from your site. Upgrade to Pro to remove the WordPress.com footer credit.`
- `Enrich your posts and pages with video or audio. Upload plenty of media, directly to your site — the Pro Plan has 50 GB storage.`
- `Pro plan required`
- `If you purchased %(domain)s elsewhere, you can {{a}}connect it{{/a}} with WordPress.com Pro.`
- `Only .blog domains are included with your plan, to use a different tld upgrade to a Pro plan.`
- `Upgrade to a Pro plan to unlock the power of our SEO tools!`
- `Unlock powerful SEO tools! Contact your site's administrator to upgrade to a Pro plan.`
- `Use this theme at no extra cost on our Pro Plan`
- `Only site administrators can upgrade to the Pro plan.`
- `Upgrade to Pro Plan`
- `Also included in the Pro Plan`
- `Pay an {{strong}}extra %(extraToPay)s{{/strong}} for our Pro plan, and get access to all its features, plus the first year of your domain for free.`
- `{{strong}}Save %(savings)s{{/strong}} when you purchase a WordPress.com Pro plan instead — your domain comes free for a year.`
- `Purchase our Pro plan at {{strong}}no extra cost{{/strong}}, and get access to all its features, plus the first year of your domain for free.`
- `Did you know that WordPress.com Pro plan customers can now upload any WordPress plugins they want?`
- `And access to the plugin library is just one of many benefits included with the Pro plan.`
- `With our Pro plan you can upload any custom theme you want, have full access to your site with SFTP, and have full control of your data directly with phpMyAdmin.`
- `{{b}}But please note:{{/b}} you don’t currently have access to all these amazing features because they are only available to Pro plan customers and you haven’t upgraded yet.`
- `Upgrade to the Pro plan and start earning`
- `Available only with the Pro plan.`
- `Upgrade to the Pro plan`
- `Upgrade to the Pro plan to access all hosting features`
- `No ads with WordPress.com Pro`
- `Discover an easy way to advertise your brand across Facebook and Instagram. Capture website actions to help you target audiences and measure results. {{em}}Available on Pro plan{{/em}}.`
- `Your current plan doesn't allow plugin installation. Please upgrade to Pro plan first.`
- `%d file could not be uploaded because your site does not support video files. Upgrade to a Pro plan for video support.`
- `Contact your site administrator and ask them to upgrade this site to WordPress.com Pro.`
- `Upgrade to the Pro Plan to Enable VideoPress`
- `Import your entire site with the Pro Plan.`
- `A Pro Plan is required to import everything.`
- `Yes! With the WordPress.com Pro plan you can install any theme you'd like. All plans give you access to our {{a}}directory of free and premium themes{{/a}}. These are among the highest-quality WordPress themes, hand-picked and reviewed by our team.`
- `Yes! With the WordPress.com Pro plan you can install any theme you'd like. All plans give you access to our {{a}}directory of free themes{{/a}}. These are among the highest-quality WordPress themes, hand-picked and reviewed by our team.`
- `Yes! With the WordPress.com Pro plan you can search for and install external plugins. All plans already come with a custom set of plugins tailored just for them. {{a}}Find out more about plugins{{/a}}.`
- `Included in the Pro plan (%s):`
- `Upgrade to the Pro plan to install plugins.`
- `No Ads with WordPress.com Pro`
- `Available with Pro plan`
- `Comes built-in with WordPress.com Pro plan.`
- `Available on the Pro plan`
- `Remove the footer credit entirely with WordPress.com Pro`
- `Upload Audio with WordPress.com Pro`
- `Access this theme for FREE with a Pro plan!`
- `Unlock ALL premium themes with our Pro plan!`
- `Upload your own themes with our Pro plan!`
- `Upgrade to the Pro plan to access the theme install features`
- `Requires a Pro plan.`
- `You'll need the Pro plan.`
- `Requires a {{a}}Pro plan{{/a}}`
- `Grow your business with <strong>WordPress.com Pro</strong>`
- `Monetize your site with <strong>WordPress.com Pro</strong>`
- `By upgrading to the Pro plan, you'll enable audio upload support on your site.`
- `Available with the Pro plan`

Thank you in advance!

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7175355

Hi @southp, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include all of the following strings:
- `Along with all the tools and features built right into WordPress.com, the Pro plan allows you to install other plugins.`
- `When you want to build a one-of-a-kind website, it's time for WordPress.com Pro: upload plugins and themes to create a truly tailored experience for your visitors.`
- `Sites on the Pro Plan using custom plugins and/or custom themes now have the option to switch PHP versions.`
- `All WordPress.com advertising has been removed from your site. Upgrade to Pro to remove the WordPress.com footer credit.`
- `Enrich your posts and pages with video or audio. Upload plenty of media, directly to your site — the Pro Plan has 50 GB storage.`
- `Pro plan required`
- `If you purchased %(domain)s elsewhere, you can {{a}}connect it{{/a}} with WordPress.com Pro.`
- `Only .blog domains are included with your plan, to use a different tld upgrade to a Pro plan.`
- `Upgrade to %(planName)s to unlock the power of our SEO tools!`
- `Unlock powerful SEO tools! Contact your site's administrator to upgrade to %(planName)s.`
- `Use this theme at no extra cost on our Pro Plan`
- `Only site administrators can upgrade to the Pro plan.`
- `Upgrade to Pro Plan`
- `Also included in the Pro Plan`
- `Pay an {{strong}}extra %(extraToPay)s{{/strong}} for our Pro plan, and get access to all its features, plus the first year of your domain for free.`
- `{{strong}}Save %(savings)s{{/strong}} when you purchase a WordPress.com Pro plan instead — your domain comes free for a year.`
- `Purchase our Pro plan at {{strong}}no extra cost{{/strong}}, and get access to all its features, plus the first year of your domain for free.`
- `Did you know that WordPress.com Pro plan customers can now upload any WordPress plugins they want?`
- `And access to the plugin library is just one of many benefits included with the Pro plan.`
- `With our Pro plan you can upload any custom theme you want, have full access to your site with SFTP, and have full control of your data directly with phpMyAdmin.`
- `{{b}}But please note:{{/b}} you don’t currently have access to all these amazing features because they are only available to Pro plan customers and you haven’t upgraded yet.`
- `Upgrade to the Pro plan and start earning`
- `Upgrade to the Pro plan`
- `Upgrade to the Pro plan to access all hosting features`
- `No ads with WordPress.com Pro`
- `Discover an easy way to advertise your brand across Facebook and Instagram. Capture website actions to help you target audiences and measure results. {{em}}Available on Pro plan{{/em}}.`
- `Your current plan doesn't allow plugin installation. Please upgrade to Pro plan first.`
- `%d file could not be uploaded because your site does not support video files. Upgrade to a Pro plan for video support.`
- `Contact your site administrator and ask them to upgrade this site to WordPress.com Pro.`
- `Upgrade to the Pro Plan to Enable VideoPress`
- `Import your entire site with the Pro Plan.`
- `A Pro Plan is required to import everything.`
- `Yes! With the WordPress.com Pro plan you can install any theme you'd like. All plans give you access to our {{a}}directory of free and premium themes{{/a}}. These are among the highest-quality WordPress themes, hand-picked and reviewed by our team.`
- `Yes! With the WordPress.com Pro plan you can install any theme you'd like. All plans give you access to our {{a}}directory of free themes{{/a}}. These are among the highest-quality WordPress themes, hand-picked and reviewed by our team.`
- `Yes! With the WordPress.com Pro plan you can search for and install external plugins. All plans already come with a custom set of plugins tailored just for them. {{a}}Find out more about plugins{{/a}}.`
- `Included in the Pro plan (%s):`
- `Upgrade to the Pro plan to install plugins.`
- `No Ads with WordPress.com Pro`
- `Comes built-in with the WordPress.com Pro plan.`
- `Available on the Pro plan`
- `Remove the footer credit entirely with WordPress.com Pro`
- `Upload Audio with WordPress.com Pro`
- `Access this theme for FREE with a Pro plan!`
- `Unlock ALL premium themes with our Pro plan!`
- `Upload your own themes with our Pro plan!`
- `Requires a Pro plan.`
- `You'll need the Pro plan.`
- `Grow your business with <strong>WordPress.com Pro</strong>`
- `Monetize your site with <strong>WordPress.com Pro</strong>`
- `By upgrading to the Pro plan, you'll enable audio upload support on your site.`
- `Available with the Pro plan`
- `a Pro plan`

Thank you in advance!

File: client/my-sites/earn/ads/wrapper.jsx [hunk 1]
```diff
@@ -201,14 +201,14 @@ class AdsWrapper extends Component {
 
 	renderUpsell() {
 		const { siteSlug, translate } = this.props;
-		const bannerURL = `/checkout/${ siteSlug }/premium`;
+		const bannerURL = `/checkout/${ siteSlug }/pro`;
```
Comment (southp) [inline client/my-sites/earn/ads/wrapper.jsx:204]: This has created a path for a legacy plan to transit to the Pro plan, but we will leave it for now: p2-pdgrnI-xR#comment-1052

File: client/my-sites/hosting/main.js [hunk 2]
```diff
@@ -72,10 +72,10 @@ class Hosting extends Component {
 
 		const getUpgradeBanner = () => (
 			<UpsellNudge
-				title={ translate( 'Upgrade to the Business plan to access all hosting features' ) }
+				title={ translate( 'Upgrade to the Pro plan to access all hosting features' ) }
```
Comment (southp) [inline client/my-sites/hosting/main.js:75]: This has created a path for a legacy plan to transit to the Pro plan, but we will leave it for now: p2-pdgrnI-xR#comment-1052

File: client/my-sites/marketplace/pages/marketplace-plugin-install/index.tsx [hunk 3]
```diff
@@ -264,10 +264,10 @@ const MarketplacePluginInstall = ( {
 					illustration="/calypso/images/illustrations/error.svg"
 					title={ null }
 					line={ translate(
-						"Your current plan doesn't allow plugin installation. Please upgrade to Business plan first."
+						"Your current plan doesn't allow plugin installation. Please upgrade to Pro plan first."
```
Comment (southp) [inline client/my-sites/marketplace/pages/marketplace-plugin-install/index.tsx:267]: This has created a path for a legacy plan to transit to the Pro plan, but we will leave it for now: p2-pdgrnI-xR#comment-1052

File: client/my-sites/scan/wpcom-upsell.tsx [hunk 4]
```diff
@@ -77,17 +77,17 @@ export default function WPCOMScanUpsellPage(): ReactElement {
 				</p>
 				<PromoCardCTA
 					cta={ {
-						text: translate( 'Upgrade to Business Plan' ),
+						text: translate( 'Upgrade to Pro Plan' ),
```
Comment (southp) [inline client/my-sites/scan/wpcom-upsell.tsx:80]: This has created a path for a legacy plan to transit to the Pro plan, but we will leave it for now: p2-pdgrnI-xR#comment-1052

File: client/my-sites/theme/main.jsx [hunk 5]
```diff
@@ -728,9 +727,9 @@ class ThemeSheet extends Component {
 		if ( hasWpComThemeUpsellBanner ) {
 			pageUpsellBanner = (
 				<UpsellNudge
-					plan={ PLAN_PREMIUM }
+					plan={ PLAN_WPCOM_PRO }
```
Comment (southp) [inline client/my-sites/theme/main.jsx:730]: This has created a path for a legacy plan to transit to the Pro plan, but we will leave it for now: p2-pdgrnI-xR#comment-1052

File: client/components/seo/preview-upgrade-nudge/index.jsx [hunk 6]
```diff
@@ -38,14 +38,14 @@ export const SeoPreviewNudge = ( {
 					site &&
 					findFirstSimilarPlanKey(
 						site.plan.product_slug,
-						isJetpack ? { type: TYPE_SECURITY_DAILY, term: TERM_ANNUALLY } : { type: TYPE_BUSINESS }
+						isJetpack ? { type: TYPE_SECURITY_DAILY, term: TERM_ANNUALLY } : { type: TYPE_PRO }
```
Comment (southp) [inline client/components/seo/preview-upgrade-nudge/index.jsx]: This has created a path for a legacy plan to transit to the Pro plan, but we will leave it for now: p2-pdgrnI-xR#comment-1052

File: client/my-sites/site-settings/cloudflare.js [hunk 7]
```diff
@@ -43,7 +43,7 @@ const Cloudflare = () => {
 								<p className="site-settings__cloudflare-title">
 									{ translate( 'Jetpack Site Accelerator' ) }
 								</p>
-								<p>{ translate( 'Comes built-in with WordPress.com Business plans.' ) }</p>
+								<p>{ translate( 'Comes built-in with WordPress.com Pro plan.' ) }</p>
```
Comment (claudiucelfilip) [inline client/my-sites/site-settings/cloudflare.js]: Should probably have a `the` before WordPress.com Pro

File: client/my-sites/plans-features-main/wpcom-faq.jsx [hunk 8]
```diff
@@ -40,7 +40,7 @@ const WpcomFAQ = ( { isChatAvailable, siteSlug, translate } ) => {
 			<FAQItem
 				question={ translate( 'Do you sell domains?' ) }
 				answer={ translate(
-					'Yes! Annual and biannual Personal, Premium, Business, and eCommerce plans include a free custom domain for one year. ' +
+					'Yes! Annual and biennial Pro plan include a free custom domain for one year. ' +
```
Comment (claudiucelfilip) [inline client/my-sites/plans-features-main/wpcom-faq.jsx]: Should probably say `Yes! Pro plan includes a free ...` since there's only the annual Pro plan

Comment (southp) [reply client/my-sites/plans-features-main/wpcom-faq.jsx]: Great catch. Done in https://github.com/Automattic/wp-calypso/pull/61595/commits/f285d0dffce8efcc20021d135797fcc0620d2e8c

File: client/my-sites/site-settings/cloudflare.js (hunk 7 above)
Comment (southp) [reply client/my-sites/site-settings/cloudflare.js]: Fixed by https://github.com/Automattic/wp-calypso/pull/61595/commits/e97f4cdc8360039da0ffde3243130689c5869ca3

**Comment by southp** [comment]
[The style check fail](https://teamcity.a8c.com/viewLog.html?buildId=7663704&tab=Inspection&buildTypeId=calypso_CheckCodeStyleBranch) isn't related to this PR, so I'll ignore it for now:
image

**Review by claudiucelfilip — APPROVED** [review]
LGTM 🚀

**Comment by emilyaudela** [comment]
The following string doesn't make sense because it references 2 different plan levels.

`With your plan, all [WordPress.com](http://wordpress.com/) advertising has been removed from your site. You can upgrade to a Pro plan to also remove the [WordPress.com](http://wordpress.com/) footer credit.`

It should be changed to:

`With your Pro plan, all [WordPress.com](http://wordpress.com/) advertising, including the WordPress.com footer credit has been removed from your site.`

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7175355).

**Pull request merged**